src/plugins/data/common/search/search_source/search_source.ts

```diff
@@ -270,16 +278,16 @@ export class SearchSource {
             return acc.concat(this.parseActiveIndexPatternFromQueryString(currStr));
           }, []) ?? [];
 
-    const activeIndexPattern: string[] = filters?.reduce<string[]>((acc, f) => {
-      if (f.meta.key === '_index' && f.meta.disabled === false) {
-        if (f.meta.negate === false) {
-          return _.concat(acc, f.meta.params.query ?? f.meta.params);
-        } else {
-          if (Array.isArray(f.meta.params)) {
-            return _.difference(acc, f.meta.params);
+    const activeIndexPattern = filters?.reduce((acc, f) => {
+      if (isPhraseFilter(f)) {
```

Comment:
I found this makes the index filters missing from the inspect flyout. As the structure of phrase filter here is different from the `isPhraseFilter`:

<img width="2531" alt="Screenshot 2023-02-13 at 14 52 00" src="https://user-images.githubusercontent.com/6295984/218492841-b9fca7d5-b084-4823-b9c7-405d7eb545f6.png">